movo_demos/launch/nav/sensor_nav.launch

```diff
@@ -6,7 +6,7 @@
     <arg name="local" default="true"/>
 
     <machine name="movo1" address="movo1" user="robohub" 
-                 env-loader="/home/movo/env.sh" default="true" unless="$(arg local)"/>
+                 env-loader="/home/robohub/env.sh" default="true" unless="$(arg local)"/>
```

Comment:
Need to check what these env.sh files are on the robot unless @le2tang do you have any ideas?

Reply:
It's a simple script that sources noetic and the workspace that gets called when a terminal is opened on either of the movo computers